src/handlers/payout/action.ts

```diff
@@ -189,7 +190,14 @@ export const handleIssueClosed = async () => {
   logger.info(`Posting a payout url to the issue, url: ${payoutUrl}`);
   const comment =
     `#### Task Assignee Reward\n### [ **[ CLAIM ${priceInEth} ${tokenSymbol.toUpperCase()} ]** ](${payoutUrl})\n` + "```" + shortenRecipient + "```";
-  const permitComments = comments.filter((content) => content.body.includes("https://pay.ubq.fi?claim=") && content.user.type == UserType.Bot);
+  const permitComments = comments.filter((content) => {
```

Comment:
extract this into helper function and apply it everywhere where we search for permit

Reply:
If we make it as a helper fn, that would be cool but not so important for now. To speed up the velocity, mark it as resolved